
sers
time data '' does not match format '%Y-%m-%d %H:%M:%S'Traceback (most recent call last):
  File "C:/Users/rossh/Documents/PHY346/PHY346_NationalGrid_2019/sauce/Filter_15min_merge_test.py", line 28, in indi_HP_table
    dt = datetime.strptime(row[0], "%Y-%m-%d %H:%M:%S")
  File "C:\Users\rossh\Anaconda3\lib\_strptime.py", line 577, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "C:\Users\rossh\Anaconda3\lib\_strptime.py", line 359, in _strptime
    (data_string, format))
ValueError: time data '' does not match format '%Y-%m-%d %H:%M:%S'

 ============================================================
sers
time data '' does not match format '%Y-%m-%d %H:%M:%S'Traceback (most recent call last):
  File "C:/Users/rossh/Documents/PHY346/PHY346_NationalGrid_2019/sauce/Filter_15min_merge_test.py", line 28, in indi_HP_table
    dt = datetime.strptime(row[0], "%Y-%m-%d %H:%M:%S")
  File "C:\Users\rossh\Anaconda3\lib\_strptime.py", line 577, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "C:\Users\rossh\Anaconda3\lib\_strptime.py", line 359, in _strptime
    (data_string, format))
ValueError: time data '' does not match format '%Y-%m-%d %H:%M:%S'

 ============================================================
sers
time data '' does not match format '%Y-%m-%d %H:%M:%S'Traceback (most recent call last):
  File "C:/Users/rossh/Documents/PHY346/PHY346_NationalGrid_2019/sauce/Filter_15min_merge_test.py", line 28, in indi_HP_table
    dt = datetime.strptime(row[0], "%Y-%m-%d %H:%M:%S")
  File "C:\Users\rossh\Anaconda3\lib\_strptime.py", line 577, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "C:\Users\rossh\Anaconda3\lib\_strptime.py", line 359, in _strptime
    (data_string, format))
ValueError: time data '' does not match format '%Y-%m-%d %H:%M:%S'

 ============================================================
sers
time data '' does not match format '%Y-%m-%d %H:%M:%S'Traceback (most recent call last):
  File "C:/Users/rossh/Documents/PHY346/PHY346_NationalGrid_2019/sauce/Filter_15min_merge_test.py", line 28, in indi_HP_table
    dt = datetime.strptime(row[0], "%Y-%m-%d %H:%M:%S")
  File "C:\Users\rossh\Anaconda3\lib\_strptime.py", line 577, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "C:\Users\rossh\Anaconda3\lib\_strptime.py", line 359, in _strptime
    (data_string, format))
ValueError: time data '' does not match format '%Y-%m-%d %H:%M:%S'

 ============================================================
sers
time data '' does not match format '%Y-%m-%d %H:%M:%S'Traceback (most recent call last):
  File "C:/Users/rossh/Documents/PHY346/PHY346_NationalGrid_2019/sauce/Filter_15min_merge_test.py", line 28, in indi_HP_table
    dt = datetime.strptime(row[0], "%Y-%m-%d %H:%M:%S")
  File "C:\Users\rossh\Anaconda3\lib\_strptime.py", line 577, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "C:\Users\rossh\Anaconda3\lib\_strptime.py", line 359, in _strptime
    (data_string, format))
ValueError: time data '' does not match format '%Y-%m-%d %H:%M:%S'

 ============================================================
sers
time data '' does not match format '%Y-%m-%d %H:%M:%S'Traceback (most recent call last):
  File "C:/Users/rossh/Documents/PHY346/PHY346_NationalGrid_2019/sauce/Filter_15min_merge_test.py", line 28, in indi_HP_table
    dt = datetime.strptime(row[0], "%Y-%m-%d %H:%M:%S")
  File "C:\Users\rossh\Anaconda3\lib\_strptime.py", line 577, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "C:\Users\rossh\Anaconda3\lib\_strptime.py", line 359, in _strptime
    (data_string, format))
ValueError: time data '' does not match format '%Y-%m-%d %H:%M:%S'

 ============================================================